src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -313,6 +316,19 @@ public void setMaxScore(Double maxScore) {
         this.maxScore = maxScore;
     }
 
+    public Double getBonusPoints() {
+        return bonusPoints;
+    }
+
+    public Exercise bonusPoints(Double bonusPoints) {
```

Comment:
Should we add some documentation for what this method should be used? The return type and name of the method are not clear

Reply:
this is the normal convenience setter that we use in all domain classes. It allows you to write

`exercise.bonusPoints(2.0).bonusMultiplier(3.0)...`

instead of

```
exercise.setBonusPoints(2.0);
exercise.setBonusMultiplier(3.0);
...
```